src/Options.php

```diff
@@ -571,6 +573,19 @@ public static function setInputDefinition(InputDefinition $inputDefinition): voi
                 InputOption::VALUE_NONE,
                 '@see PHPUnit guide, chapter: ' . $chapter,
             ),
+            new InputOption(
+                'enforce-time-limit',
+                null,
+                InputOption::VALUE_NONE,
+                '@see PHPUnit guide, chapter: ' . $chapter,
```

Comment:
The chapter is wrong: please move it between `disallow-test-output` and `dont-report-useless-tests`